app/scripts/modules/core/src/plugins/plugin.registry.ts

```diff
@@ -168,6 +169,9 @@ export class PluginRegistry {
       // Register extensions with deck.
       plugin.stages?.forEach(stage => Registry.pipeline.registerStage(stage));
 
+      // Run code that currently does not have an extension point
+      plugin.initialize && plugin.initialize();
```

Comment:
Some fancy new TypeScript sugar:
```suggestion
      plugin.initialize?.();
```